build/common/installer/scripts/tomlparser-geneva-config.rb

```diff
@@ -289,42 +310,46 @@ def is_configure_geneva_env_vars()
   file = File.open("setgenevaconfigenv.ps1", "w")
 
   if !file.nil?
-    commands = get_command_windows("GENEVA_LOGS_INTEGRATION", @geneva_logs_integration)
-    file.write(commands)
-    commands = get_command_windows("GENEVA_LOGS_MULTI_TENANCY", @multi_tenancy)
-    file.write(commands)
-    if @enable_fbit_threading
-      commands = get_command_windows("ENABLE_FBIT_THREADING", @enable_fbit_threading)
-      file.write(commands)
-    end
-
-    if is_configure_geneva_env_vars()
-      commands = get_command_windows("MONITORING_GCS_ENVIRONMENT", @geneva_account_environment)
-      file.write(commands)
-      commands = get_command_windows("MONITORING_GCS_NAMESPACE", @geneva_account_namespace_windows)
-      file.write(commands)
-      commands = get_command_windows("MONITORING_GCS_ACCOUNT", @geneva_account_name)
+    if @disable_windows
+      puts "config::geneva_logs::info:geneva logs integration disabled for windows"
+    else
+      commands = get_command_windows("GENEVA_LOGS_INTEGRATION", @geneva_logs_integration)
```

Comment:
Same as above

Reply:
> Same as above
if disabled, none of the GENEVA related env var being set.